### DISCUSSION TITLE
zh/guide/advanced/s3

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#336] [Announcements]
# zh/guide/advanced/s3

此功能是将AList映射为S3协议，可以使用 S3客户端 (#s3-客户端推荐-填写示例) 使用和查看，就像AList可以映射为WebDav一样 可以是整个AList（根目录），也可以是单独某个文件夹; 1.使用方式 1.1 如果是第一次使用，先打开AList后台生成一下 和 保存，然后停止AList，打开配置文件启用S3服务 如果你后续更新访问ID和...

https://alist.nn.ci/zh/guide/advanced/s3.html

**Comment by ray5378** [comment]
没有  Region 的配置选项

**Reply by ray5378** [reply]
群晖的Cloud sync自定义S3无法连接

**Comment by wy16W2pIilK1xgqN** [comment]
S3 服务和WebDAV服务，性能对比如何？
那个占用少，性能高？

**Comment by eng98** [comment]
MountainDuck 可以支持吗，目前可以连接上，但是打开列表失败

**Comment by shuanglu1** [comment]
可以支持下新建文件和文件夹嘛，谢谢

**Comment by scjean** [comment]
S3 Browser只可以下载，单独下载一个文件的时候，会带上所在的文件夹。上传文件和新建文件夹无法操作。

另外Air Live Drive和AirExplorer支持S3，能列出文件夹和文件，无法进行任何操作。

**Reply by scjean** [reply]
AirExplorer上传需要专业版
Air Live Drive可以进行上传下载，需要打开SSL

**Comment by starssr** [comment]
EndPoint *
是什么呀，s3是单独的端口需要再反向代理吗
通用的配置 有示范吗

**Comment by Sebs-Lee** [comment]
ubuntu s3cmd(apt install s3cmd)
github仓库地址：https://github.com/s3tools/s3cmd

配置命令：s3cmd --configure
Access Key: ...
Secret Key: ...
Default Region: US
S3 Endpoint: 127.0.0.1:5246
DNS-style bucket+hostname:port template for accessing a bucket: 127.0.0.1/%(bucket)
Encryption password: 可以留空
Path to GPG program: 默认
Use HTTPS protocol: 开了ssl就Yes, 没开写No
HTTP Proxy server name: ...
Test access with supplied credentials: Y

> 如果配置正确会输出：
Please wait, attempting to list all buckets...
Success. Your access key and secret key worked fine :-)

Now verifying that encryption works...
Not configured. Never mind.

>测试配置，如果成功能看见alist的桶
```
s3cmd -ls
```

**Reply by starssr** [reply]
好的 我试试

**Reply by starssr** [reply]
这是Windows吗  Linux在配置文件修改了  之后怎么测试是否成功，目前使用通用s3无法链接
![image](https://github.com/user-attachments/assets/c156c9c9-172e-4f99-ab85-937d8855b245)

**Reply by Sebs-Lee** [reply]
修改配置文件之后要重启服务的，具体看你alist的部署环境，重启容器或者systemctl restart吧，至于通用s3是啥？

**Comment by MCWXT** [comment]
[![pA3nnbV.png](https://s21.ax1x.com/2024/10/01/pA3nnbV.png)](https://imgse.com/i/pA3nnbV)

有没有大佬知道这是什么情况？
